diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -316,13 +316,9 @@ int DifferentialFunctionComparator::cmpOperations(
     // values, treat them as equal for now. They will be compared at the end,
     // after all their incoming values have been compared and matched.
     if (isa<PHINode>(L) && isa<PHINode>(R)) {
-        auto PhiL = dyn_cast<PHINode>(L);
-        auto PhiR = dyn_cast<PHINode>(R);
-        if (PhiL->getNumIncomingValues() == PhiR->getNumIncomingValues()) {
-            needToCmpOperands = false;
-            phisToCompare.emplace_back(PhiL, PhiR);
-            return 0;
-        }
+        needToCmpOperands = false;
```

Comment:
The comment above should also be updated (If PHI nodes are compared and they have the same number of incoming values...).